usr/sbin/pkcscca/pkcscca.c

```diff
@@ -50,6 +53,10 @@ const char model[] = "CCA";
 const char descr[] = "IBM CCA Token";
 const char label[] = "ccatok";
 
+#if defined(_AIX)
+char *__progname = "pkcscca";
```

Comment:
should all the `__progname` variables be const ? I guess this applies to lost of placces in other commits, too.